vite.config.ts

```diff
@@ -2,21 +2,25 @@ import react from '@vitejs/plugin-react'
 import { configDotenv } from 'dotenv'
 import path from 'node:path'
 import { defineConfig } from 'vite'
-import { viteExternalsPlugin } from 'vite-plugin-externals'
+import { cdn } from 'vite-plugin-cdn2'
+import { unpkg } from 'vite-plugin-cdn2/url.js'
 
 configDotenv()
 
 // https://vitejs.dev/config/
-export default defineConfig(async env => ({
+export default defineConfig(async _=> ({
```

Comment:
This code style seems to be problematic.

The code style of this seems to be problematic, but I have a problem below, so I won't suggest here.

Reply:
I can remove it.

Reply:
```suggestion
export default defineConfig(async _ => ({
```